### PR TITLE
You can now use banhammers as a weapon

### PR DESCRIPTION
**Pull request by ghost**
BWOINK UR DED

:cl: More Robust Than You
fix: You can now use banhammers as a weapon
/:cl:

### CODE REVIEW

**Comment by praisenarsie** [comment]
a banhammer gets spawn once a blue moon.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -20,10 +20,11 @@
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == "head")
 		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
```
Comment (DoctoreMad) [inline code/game/objects/items/weaponry.dm]: >Bangammer

How did everyone miss this

Comment (optimumtact) [reply code/game/objects/items/weaponry.dm]: It's deliberate

Comment (optimumtact) [reply code/game/objects/items/weaponry.dm]: https://github.com/tgstation/tgstation/pull/30173

**Comment by Bawhoppen** [comment]
@praisenarsie You can find them in multiple ruins

**Comment by BeeSting12** [comment]
👎 Make ban hammers actually ban people

**Comment by Bawhoppen** [comment]
I think if you suicide with them it actually does ban you. Or atleast that was a PR once.

**Review by Cyberboss — APPROVED** [review]

**Review by Cyberboss — CHANGES_REQUESTED** [review]
This is useless as banhammers have no force

**Pull request merged**